packages/wonder-blocks-toolbar/docs.md

```diff
@@ -0,0 +1,192 @@
+> TODO(scottgrant): Finish building, and write more thorough docs.
+
+## Content toolbar
+
+### Standard view
+
+```js
+<Toolbar
+	contentKind="Video"
+	showClose={true}
+	title="Counting with small numbers"
+/>
+```
+
+### Content completed view
+
+```js
+const Button = require("@khanacademy/wonder-blocks-button").default;
+
+const startExercise = <Button>Next Video</Button>;
+
+<Toolbar
+	contentKind="Video"
+	rightContent={[startExercise]}
```

Comment:
I wonder if we should allow people to pass either an array or a single element.  Having to wrap single items in an array adds friction.

Reply:
John mentioned this too. I've updated the component to accept a single component or an array now, and handle that appropriately in `renderContent`.